auth0/resource_auth0_client.go

```diff
@@ -220,9 +217,7 @@ func readClient(d *schema.ResourceData, m interface{}) error {
 	if err != nil {
 		return err
 	}
-	d.SetId(c.ClientID)
-	d.Set("client_id", c.ClientID)
-	d.Set("client_secret", c.ClientSecret)
+	buildClientSchema(c, d)
```

Comment:
Lets just merge the contents of `buildClientSchema` in here since `readClient` essentially does that.